### DISCUSSION TITLE
Creating users only with $id

### DISCUSSION THREAD

**Discussion by alitvinenko** [appwrite/appwrite#3123] [General]
I using Appwrite like as additional Database. Users data storaging in my primary databse.
Authorization is implemented by the tools of my service and not through Appwrite. Therefore, I cannot create appwrite users by specifying an email and password, but I want to identify them only by $id (creating an anonymous session does not suit me since I cannot create an anonymous user with the necessary $id).

Why not modify the existing one or create a new server api method that would create a record about the user with the ability to pass his $id?

May be exists any more solution my problem?

Right now I'm creating users in Appwrite on my backend server with a fake email address and password, but I don't like this solution.

**Comment by rlee1990** [comment]
@alitvinenko can't you just use your backend to add the data for each user and associate their userId in the Appwrite database?

**Reply by maeddin** [reply]
In this case, it no longer makes sense to use AppWrite.

**Comment by gewenyu99** [comment]
Why don't you try the Server SDKs. This allows your backend services to just make a collection of $ids. Since you don't need authentication, the users in Appwrite can be just a plain collection of documents.

**Reply by maeddin** [reply]
That is not the point of this feature. The point is that users can access the database via the client SDK. Otherwise it wouldn't make sense to take AppWrite for that.
